modules/apps/web-experience/export-import/export-import-api/src/main/java/com/liferay/exportimport/staged/model/repository/StagedModelRepository.java

```diff
@@ -40,7 +40,13 @@ public void deleteStagedModel(
 	public void deleteStagedModels(PortletDataContext portletDataContext)
 		throws PortalException;
 
-	public T fetchMissingReference(String uuid, long groupId);
+	/**
+	 * @deprecated As of 2.2.0
+	 */
+	@Deprecated
```

Comment:
@mtambara why did you deprecate this?

Reply:
@matethurzo I deprecated both of these methods because there were no usages outside of tests in our codebase. Do we expect customers to implement these methods?

Reply:
@mtambara both of them are being used in the BaseStagedModelDataHandler in export-import-api so yes these are being used: https://github.com/matethurzo/liferay-portal/blob/master/modules/apps/web-experience/export-import/export-import-api/src/main/java/com/liferay/exportimport/data/handler/base/BaseStagedModelDataHandler.java#L82

can you please roll the deprecations back?